model/riscv_crypto_entropy_source.sail

```diff
@@ -0,0 +1,90 @@
+
+/*
+ * file: riscv_crypto_entropy_source.sail
+ *
+ * Contains a basic model for the RISC-V cryptography extension entropy
+ * source. See section 4 of the specifications for details.
+ * - https://github.com/riscv/riscv-crypto/releases
+ *
+ */
+
+/*
+ * The mnoise CSR
+ * ------------------------------------------------------------
+ */
+
+register mnoise : xlenbits
+
+/* mnoise is defined in machine mode only */
+function clause ext_is_CSR_defined (0x7A9, Machine) = {
+    true
+}
+
+function clause ext_read_CSR(0x7A9) = {
+    Some(mnoise)
+}
+
+/* Only bit 31 of mnoise is architecturally defined. */
+function clause ext_write_CSR(0x7A9, value) = {
+    mnoise[31] = value[31];
+    Some(mnoise)
+}
+
+/* Used by mentropy to check if we are (not) in noise capture mode */
+val      in_noise_capture_mode : unit -> bool effect {rreg}
+function in_noise_capture_mode() = {
+    if(bit_to_bool(mnoise[31])) then true else false
```

Comment:
```suggestion
    if bit_to_bool(mnoise[31]) then true else false
```
but you should make mnoise a bitfield so you can reference its fields by name rather than index (see things like mstatus).

Reply:
Okay, I'll convert `mnoise` to a bitfield, thanks for the pointer. I'll add the brackets too.